robot/Cumulus/resources/NPSP.py

```diff
@@ -1242,7 +1243,18 @@ def randomString(self,stringLength=10):
         letters = string.ascii_lowercase
         return ''.join(random.choice(letters) for i in range(stringLength))
 
-    def setupdata(self, name, contact_data=None, opportunity_data=None, account_data=None, engagement_data=None, task_data=None):
+    def scroll_button_into_view_and_click_using_js(self, type, value):
```

Comment:
Can you please add documentation to this function?

Reply:
Done